ansible/roles/test/tasks/acltb_test.yml

```diff
@@ -0,0 +1,82 @@
+#-----------------------------------------
+# Run ACL test and Perform log analysis.
+#-----------------------------------------
+
+# Pre-check testbed_type value
+- fail: msg="testbed_type is not defined."
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{testbed_type}} is invalid."
+  when: testbed_type not in ['t1-lag', 't1']
+
+- include_vars: "vars/topo_{{testbed_type}}.yml"
+
+# Gather minigraph facts
+- name: Gathering minigraph facts about the device
+  minigraph_facts: host={{ inventory_hostname }}
+  become: no
+  connection: local
+
+- name: Read port reverse alias mapping
+  set_fact:
+    alias_reverse_map: "{{ lookup('file', 'roles/sonicv2/files/ssw/{{ sonic_hwsku }}/alias_reverse_map.json') | from_json }}"
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['spine']}}"
+  when: testbed_type in ['t1', 't1-lag']
+
+# Generate file with switch information
+- template: src=acltb.j2 dest=/tmp/acltb_switch_info.txt
+  connection: local
+
+- name: Copy switch info file to the PTF host
+  copy: src=/tmp/acltb_switch_info.txt dest=/tmp/acltb_switch_info.txt
+  delegate_to: "{{ ptf_host }}"
+
+- set_fact:
+    testname: acl
+    run_dir: /tmp
+    out_dir: /tmp/ansible-loganalyzer-results
+    test_match_file: acltb_match_messages.txt
+    test_ignore_file: acltb_ignore_messages.txt
+    test_expect_file: acltb_expect_messages.txt
```

Comment:
In some loganalyzer includes
roles/test/files/tools/loganalyzer/loganalyzer_init.yml, roles/test/files/tools/loganalyzer/loganalyzer_analyze.yml